config/rbac/role.yaml

```diff
@@ -206,6 +206,7 @@ rules:
   resources:
   - validatingwebhookconfigurations
   verbs:
+  - '*'
```

Comment:
The list and watch below are redundant now

Reply:
Thanks! Fixed 👍

Reply:
Strange it seems that the `generate_code` job want those redundant lines. They do appear when running `make generate manifests`.